### PR TITLE
Dashrews/ROX-10936 baseline gen duration env

### PR DESCRIPTION
**Pull request by dashrews78**
## Description

Updated the env vars for the tests such that they use the smaller observation window for processing baselines.  Improved the timing of a couple of the tests such that they can tolerate slower environments better.

## Checklist
- [x] Investigated and inspected CI test results
- [ ] ~~Unit test and regression tests added~~
- [ ] ~~Evaluated and added CHANGELOG entry if required~~
- [ ] ~~Determined and documented upgrade steps~~
- [ ] ~~Documented user facing changes (create PR based on [openshift/openshift-docs]~~(https://github.com/openshift/openshift-docs) and merge into [rhacs-docs](https://github.com/openshift/openshift-docs/tree/rhacs-docs))

If any of these don't apply, please comment below.

## Testing Performed

TODO(replace-me)
Use this space to explain how you tested your PR, or, if you didn't test it, why
you did not do so. Valid reasons include, for example, "CI is sufficient",
"No testable changes". Feel free to attach JSON snippets, curl commands,
screenshots.

In addition to reviewing your code, reviewers **must** also review your testing
instructions and make sure they are sufficient.

### CODE REVIEW

**Comment by dashrews78** [comment]
[ROX-10936](https://issues.redhat.com/browse/ROX-10936)

File: qa-tests-backend/src/test/groovy/ProcessBaselinesTest.groovy [hunk 1]
```diff
@@ -122,8 +122,19 @@ class ProcessBaselinesTest extends BaseSpecification {
...
 
-        // Need to sleep so the baseline has time to lock
-        sleep 70000
+        // wait for baseline to come out of observation
+        baseline = evaluateWithRetry(30, 3) {
+            def tmpBaseline = ProcessBaselineService.getProcessBaseline(clusterId, deployment, containerName)
+            def now = System.currentTimeSeconds()
+            if (tmpBaseline.getStackRoxLockedTimestamp().getSeconds() > now) {
+                throw new RuntimeException(
+                    "Baseline ${deployment} is not out of observation yet. Baseline is ${tmpBaseline}.  Current time ${now}"
```
Comment (vikin91) [inline qa-tests-backend/src/test/groovy/ProcessBaselinesTest.groovy:131]: Maybe we could rephrase for easier reading? 
_not out of observation yet_ = _still in the observation window_

File: qa-tests-backend/src/test/groovy/ProcessBaselinesTest.groovy [hunk 2]
```diff
@@ -221,8 +232,11 @@ class ProcessBaselinesTest extends BaseSpecification {
         sleep 5000
         orchestrator.execInContainer(deployment, "pwd")
 
+        // Give the process indicators time to roll off the queue
+        sleep 60000
```
Comment (vikin91) [inline qa-tests-backend/src/test/groovy/ProcessBaselinesTest.groovy:236]: (optional) Would be cool to replace fixed sleep with evaluation whether the expected event has happened

**Review by vikin91 — APPROVED** [review]
LGTM, but I found some potential for improvements

File: qa-tests-backend/src/test/groovy/ProcessBaselinesTest.groovy (hunk 2 above)
Comment (janisz) [reply qa-tests-backend/src/test/groovy/ProcessBaselinesTest.groovy:236]: It could be included in `waitForViolation`

Comment (dashrews78) [reply qa-tests-backend/src/test/groovy/ProcessBaselinesTest.groovy:236]: I agree.  I will add that to https://issues.redhat.com/browse/ROX-10903 when I make similar updates to the test I previously ignored.  That way I can get out of the way of these failing nightlies.

**Review by vikin91 — APPROVED** [review]
Thanks for correcting!

**Review by janisz — APPROVED** [review]

**Pull request merged**